src/main/java/school/faang/user_service/filter/recommendation/RequestRejectionReasonFilter.java

```diff
@@ -0,0 +1,20 @@
+package school.faang.user_service.filter.recommendation;
+
+import org.springframework.stereotype.Component;
+import school.faang.user_service.dto.recommendation.RequestFilterDto;
+import school.faang.user_service.entity.recommendation.RecommendationRequest;
+
+import java.util.stream.Stream;
+
+@Component
+class RequestRejectionReasonFilter implements RequestFilter {
+    @Override
+    public boolean isApplicable(RequestFilterDto filters) {
+        return filters.getRejectionReason() != null && !filters.getRejectionReason().isBlank();
```

Comment:
А isBlank() внутри уже не делает разве проверку на null?

Reply:
Ответ нет: " public boolean isBlank()
Returns true if the string is empty or contains only white space codepoints, otherwise false.
Returns:
true if the string is empty or contains only white space codepoints, otherwise false"